Assets/02. Scripts/02-06. Structure/Machine/Machine.cs

```diff
@@ -0,0 +1,49 @@
+using NUnit.Framework;
+using System.Collections.Generic;
+using UnityEngine;
+
+public abstract class Machine : MonoBehaviour, IItemContainer, IInteractable
+{
+    private MachineData _data;
+    public MachineData Data => _data;
+
+    protected float _currentProgress;
+    protected int _leftOutputAmount;
+    protected bool _isFinished;
+    protected bool _isStarted;
```

Comment:
무엇이 시작되었고 무엇이 끝난것인지 필드명이 모호한것 같습니다. 아래와 같이 바꾸면 어떨까요?
```suggestion
    protected bool _isProcessFinished;
    protected bool _isProcessStarted;
```

Reply:
확인했습니다.